uui/components/layout/VirtualList.tsx

```diff
@@ -81,6 +81,7 @@ export const VirtualList = React.forwardRef<ScrollbarsApi, VirtualListProps>((pr
             onScroll={ handleScroll }
             renderView={ ({ style }: any) => <VirtualListView isLoading={ props.isLoading } style={ style } /> }
             ref={ scrollBarsRef }
+            rawProps={ props.rawProps }
```

Comment:
I guess that we may not to pass rawProps to the `ScrollBars`, due to we have VirtualListView component which is a wrapper for virtual list content. This component already has `rawProps` prop, but it's needed to pass them in 82 line.

Reply:
Yeah, I thought about that in the first place, but I didn't like that element with `class=table` is not receiving `role=table`, but its' child does

Reply:
I think it's not a problem. Moreover, I supposed that the second solution will be more correct, because it that case role=table will be on wrapper which directly contains header and rows